syntax = "proto2";
import "goby/middleware/protobuf/frontseat_config.proto";
import "goby/protobuf/option_extensions.proto";

package goby.middleware.frontseat.protobuf;

message IverConfig
{
    required string serial_port = 1;
    optional uint32 serial_baud = 2 [default = 19200];
    // write GPS feed to this serial port for NTP to use
    optional string ntp_serial_port = 3;
    optional int32 max_pitch_angle_degrees = 4 [default = 45];
}

extend Config
{
    optional IverConfig iver_config = 1003;
}
